### PR TITLE
py-2.3.0 release notes

### PR DESCRIPTION
**Pull request by jkiang13**
Release notes for v2.3.0

static html can be previewed here: https://jkiang13.github.io/synapsePythonClient/build/html/news.html

### CODE REVIEW

File: docs/news.rst [hunk 1]
```diff
@@ -2,6 +2,81 @@
 Release Notes
 =============
 
+2.3 (2021-02-XX)
```
Comment (stacey-w-taylor) [inline docs/news.rst:5]: Verifying that when this goes live the "XX" will be updated with the correct day.

File: docs/news.rst [hunk 2]
```diff
@@ -2,6 +2,81 @@
...
+
+   # programmatic
+   syn = synapseclient.login(authToken=<token>)
+
+  .. code-block::
+
+   # command line
+   synapse login -p <token>
+
+- The location of the where downloaded entities are cached can be customized to a location other than the user's home directory.
```
Comment (stacey-w-taylor) [inline docs/news.rst:29]: Remove extra word "the" so this reads as: "The location of where downloaded entities are cached..."

File: docs/news.rst [hunk 3]
```diff
@@ -2,6 +2,81 @@
...
+-  [`SYNPY-1112 <https://sagebionetworks.jira.com/browse/SYNPY-1112>`__] -
+   Concurrent migration of entities sharing the same file handle can result in an error
+-  [`SYNPY-1114 <https://sagebionetworks.jira.com/browse/SYNPY-1114>`__] -
+   Mitigate new Rust compiler dependency on Linux via transitive cryptography dependency
+
+New Features
+------------
+
+-  [`SYNPY-1058 <https://sagebionetworks.jira.com/browse/SYNPY-1058>`__] -
+   Accept oauth access token for authentication to use synapse rest services
```
Comment (stacey-w-taylor) [inline docs/news.rst:63]: Suggest capitalizing "Synapse" and "REST"

File: docs/news.rst [hunk 4]
```diff
@@ -2,6 +2,81 @@
...
+
+Highlights
+----------
+
+- The `index_files_for_migration <synapseutils.html#synapseutils.migrate_functions.index_files_for_migration>`__ and
+  `migrate_indexed_files <synapseutils.html#synapseutils.migrate_functions.migrate_indexed_files>`__ functions are added
+  to synapseutils to help migrate files in Synapse projects and folders between AWS S3 buckets in the same region.
+  More details on using these utilities can be found `here <S3Storage.html#storage-location-migration>`__.
+
+- This version supports login programatically and from the command line using personal access tokens which can be obtained
```
Comment (stacey-w-taylor) [inline docs/news.rst:16]: Change "which" to "that". 

"tokens which can be obtained" to " tokens that can be obtained"

**Review by stacey-w-taylor — APPROVED** [review]
I left 4 in-line comments -- all minor.  Looks good!

File: docs/news.rst (hunk 1 above)
Comment (jkiang13) [reply docs/news.rst:5]: Yes, I'll update with the exact release date.

**Review by kelshmo — APPROVED** [review]

**Pull request merged**